### ISSUE TITLE
Iframe Transport not working on Safari 13

### ISSUE THREAD

**Issue by liukefu2050** [blueimp/jQuery-File-Upload#3633]
Safari 13.0.5
MacOs 10.15.3 
jquery  1.11.1
on line code:  https://ms.365me.me/index.jsp

use iframe-transport ,but got about:blank

            (
              ((aborted ||
                that._trigger(
                  'send',
                  $.Event('send', { delegatedEvent: e }),
                  options
                ) === false) &&
                that._getXHRPromise(false, options.context, aborted)) ||
              that._chunkedUpload(options) ||
              $.ajax(options)
            )
              .done(function (result, textStatus, jqXHR) {
                **// not run here**
              })
              .fail(function (jqXHR, textStatus, errorThrown) {
                **// not run here**
              })
              .always(function (jqXHRorResult, textStatus, jqXHRorError) {
              **// not run here**
              });

thanks for help!

**Comment by blueimp** [comment]
Can conform this is a bug, making the `forceIframeTransport: true` option unusable with Safari at the moment.

I've been able to create a [minimal test case](https://jsfiddle.net/px8kaLf6/) that reproduces the issue:

```js
var iframe = document.createElement('iframe')
iframe.name = 'iframe'

var button = document.createElement('button')
button.type = button.innerText = 'submit'

var form = document.createElement('form')
form.action = 'https://example.org/'
form.method = 'POST'
form.target = 'iframe'
form.appendChild(button)
form.appendChild(iframe)

iframe.onload = function () {
  iframe.onload = function () {
    alert('iframe load event fired')
  }
  form.submit()
}
iframe.src = 'javascript:false;'

document.body.appendChild(form)
```

On other browsers, this displays the alert box on load, on Safari the alert box only shows when pressing the submit button.

A [fix for the simple test case](https://jsfiddle.net/yg1onfs5/) is wrapping the submit in a `setTimeout` call:

```js
var iframe = document.createElement('iframe')
iframe.name = 'iframe'

var button = document.createElement('button')
button.type = button.innerText = 'submit'

var form = document.createElement('form')
form.action = 'https://example.org/'
form.method = 'POST'
form.target = 'iframe'
form.appendChild(button)
form.appendChild(iframe)

iframe.onload = function () {
  iframe.onload = function () {
    alert('iframe load event fired')
  }
  setTimeout(function () {
    form.submit()
  })
}
iframe.src = 'javascript:false;'

document.body.appendChild(form)
```

However there seem to be follow-up issues with Safari and the Iframe Transport, will investigate further.

As a side note, if you can access the demo, you can enable the Iframe transport with the following line executed in the browser console:

```js
$('#fileupload').fileupload('option', 'forceIframeTransport', true)
```

**Comment by blueimp** [comment]
Thanks for reporting this @liukefu2050.
It's been fixed in `blueimp-file-upload@10.13.1`.
